goverance.md

```diff
@@ -0,0 +1,67 @@
+
+## Working Group Roles
+
+The SAFE Working group aims to have three Chairs, Technical Leads for each substantive block of work, and many members.
+
+Unless otherwise stated, individuals are expected to be responsive and active within
+their roles.  Within this section "member" refers to a member of a Chair, Technical Lead or
+Members.
+
+### Role of Members
+- The primary role of a member is to contribute expertise to the group.
+- Initial members are defined at the founding of the WG.
+- To add yourself as a member, submit a PR adding yourself to the list of members.
+- Members *SHOULD* remain active and responsive in their Roles.
+- Members taking an extended leave of 1 or more months *SHOULD* coordinate with other members to ensure the role is adequately staffed during the leave.
+- Members going on leave for 1-3 months *MAY* work with other members to identify a temporary replacement.
+- Members of a role *SHOULD* remove any other members that have not communicated a leave of absence and either cannot be reached for more than 1 month or are not fulfilling their documented responsibilities for more than 1 month. This may be done through a [super-majority] vote of members, or if there are not enough *active* members to get a super-majority of votes cast, then removal may occur through a [super-majority] vote between Chairs and Technical Leads. Those roles are described below.
+- Membership disagreements may be escalated to the WG Chairs.  WG Chair membership disagreements may be escalated to the CNCF Steering Committee.
+- Members *MAY* decide to step down at anytime and propose a replacement.  
+- The group will use lazy consensus amongst other members with fallback on majority vote to accept proposal.  The candidate *SHOULD* be supported by a majority of WG Members.
+- Members *MAY* select additional members through a [super-majority] vote amongst members. This *SHOULD* be supported by a majority of WG Members.
+
+### Role of Chairs
+
+* Primary role of chairs is to run operations and processes governing the WG.
+* The Chairs are responsible for providing technical guidance, for bringing up proposals that have been submitted, and for asking for new proposals to be made.
+* A chair will serve in their role for 2 years from when they started in the role, and *MAY* run again indefinitely.
+* To fill a vacancy for a new chair, the group will announce with at least two week’s notice that a vote for chair will occur. Each company will have one vote. Any group member is eligible to run for Chair.
+* There will be three chairs and their membership will be tracked in [`wg.yaml`]().
+
+
+### Role of Technical Leads
+
+Technical Leads will lead larger streams of work that require sustained technical leadership. Any new proposal should have a TL working as an active sponsor of the proposal, guiding the proposal with the engagement of community. The general list of activities for TL are
```

Comment:
I think here we should define the abbreviation before we use it.

Suggested edit: "Technical Leads (TLs) will lead ..."

Reply:
I'll make this change.

Reply:
This is now:
> Technical Leads (TLs) will lead larger streams of work that require sustained
technical leadership. Any new proposal should have a TL working as an active
sponsor of the proposal, guiding the proposal with the engagement of community.
The general list of activities for TL are: